Lib/tkinter/__init__.py

```diff
@@ -838,6 +838,21 @@ def after_cancel(self, id):
             pass
         self.tk.call('after', 'cancel', id)
 
+    def after_info(self, id=None):
+        """Return information about existing event handlers.
+
+        With no argument, return a tuple of the identifiers for all existing
+        event handlers created by the after and after_idle commands for this
+        interpreter.  If id is supplied, it specifies an existing handler; id
+        must have been the return value from some previous call to after or
+        after_idle and it must not have triggered yet or been canceled. If the
+        id doesn't exist, a TclError is raised.  Otherwise, the return value is
+        a tuple containing (script, type) where script is a reference to the
+        function to be called by the event handler and type is either 'idle'
+        or 'timer' to indicate what kind of event handler it is.
+        """
+        return self.tk.splitlist(self.tk.call('after', 'info', id))
```

Comment:
```suggestion
        result = self.tk.splitlist(self.tk.call('after', 'info', id))
        if id is not None:
            result = tuple(map(self.tk.splitlist, result))
        return result
```
We need to split nested level tuples. Otherwise we sould get a tuple of strings when `wantobject=0`.

Please test your changes manually by temporary setting `wantobject=0` in the `tkinter` module (we need to add a CLI test option for this, but it is a different issue).